opentelemetry-stdout/examples/basic.rs

```diff
@@ -64,21 +64,92 @@ fn init_logs() -> opentelemetry_sdk::logs::LoggerProvider {
 
 #[cfg(feature = "trace")]
 fn emit_span() {
-    let tracer = global::tracer("stdout-test");
-    let mut span = tracer.start("test_span");
-    span.set_attribute(KeyValue::new("test_key", "test_value"));
+    use opentelemetry::trace::{
+        SpanContext, SpanId, TraceFlags, TraceId, TraceState, TracerProvider,
+    };
+
+    let tracer = global::tracer_provider()
+        .tracer_builder("stdout-example")
+        .with_version("v1")
+        .with_schema_url("schema_url")
+        .with_attributes(vec![KeyValue::new("scope_key", "scope_value")])
```

Comment:
Let's showcase using arrays instead of vec in examples.

```suggestion
        .with_attributes([KeyValue::new("scope_key", "scope_value")])
```